### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by veekeys**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+**Code Sandbox**
+Link to a minimal repro (fork [this code sandbox](https://codesandbox.io/s/itwinui-react-example-son74)): <!-- here -->
+
+**Steps to reproduce**
+1. <!-- fill this out -->
+1. <!-- fill this out -->
+1. <!-- fill this out -->
+
+**Actual behavior**
+A clear and concise description of what the bug is.
```
Comment (bentleyvk) [inline .github/ISSUE_TEMPLATE/bug_report.md:24]: It is shown on the same line. Better make it to move to new line.

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Is your feature request related to a problem? Please describe.**
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
```
Comment (bentleyvk) [inline .github/ISSUE_TEMPLATE/feature_request.md:11]: Here also is on the same line. Better would be in new line.

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (veekeys) [reply .github/ISSUE_TEMPLATE/bug_report.md:24]: fixed

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 2 above)
Comment (veekeys) [reply .github/ISSUE_TEMPLATE/feature_request.md]: fixed

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
+---
+name: Feature request 💄
+about: Suggest an idea for this project
+title: ''
+labels: ''
```
Comment (bentleyvk) [inline .github/ISSUE_TEMPLATE/feature_request.md:5]: No label?

Comment (veekeys) [reply .github/ISSUE_TEMPLATE/feature_request.md:5]: I dont get, where my changes dissapear

**Review by bentleyvk — APPROVED** [review]

**Review by mayank99 — APPROVED** [review]

**Pull request merged**